inst/qml/RegressionLinearBayesian.qml

```diff
@@ -263,43 +263,43 @@ Form {
 				RadioButton { value: "uniform"; label: qsTr("Uniform")}
 				RadioButton
 				{
-					value: "Wilson"
+					value: "wilson"
 					label: qsTr("Wilson")
 					childrenOnSameRow: true
 					childrenArea.columnSpacing: 1
 					DoubleField { name: "wilsonParamLambda"; label: qsTr("λ"); defaultValue: 1; inclusive: JASP.None; min: 0}
 				}
 				RadioButton
 				{
-					value: "Castillo"
+					value: "castillo"
 					label: qsTr("Castillo")
 					childrenOnSameRow: true
 					childrenArea.columnSpacing: 1
 					DoubleField { name: "castilloParamU"; label: qsTr("u"); defaultValue: 1; inclusive: JASP.MinMax; min: 1}
 				}
 				RadioButton
 				{
-					value: "Bernoulli"; label: qsTr("Bernoulli")
+					value: "bernoulli"; label: qsTr("Bernoulli")
 					childrenOnSameRow: true
 					DoubleField { name: "bernoulliParam"; label: qsTr("p"); defaultValue: 0.5; max: 1; inclusive: JASP.None; decimals: 3 }
 				}
 			}
 
 			RadioButtonGroup
 			{
-				name: "samplingMethod"
```

Comment:
I changed this because it was in the example, but I think `samplingMethod` is more clear than `sampling`. The former sounds like you need to specify a method (e.g., a string), while the latter sounds like you need to pass `TRUE` or `FALSE`.